### PR TITLE
#33-as a creater we need to be able to add an image-JustinasBrazauskasadd column for image url

### LINKED ISSUE #33: as a creater we need to be able to add an image
we can add it by inserting Image link for this  stage

we need to add URL column in the Energizer table

### PR DESCRIPTION
**Pull request by JustinasBrazauskas**
## Naming Rules

Name your PR like this: ISSUENUMBER-TITLE-YOURNAME

## Description

column for image url
## Related to

Make sure you include the issue number with a hash sign # so Github can link this PR to the right issue, like this:

Fixes #33

## Checklist:

- [ ] My code follows the style guidelines of this project
- [ ] I have carefully reviewed my own code
- [ ] I have commented my code
- [ ] I have updated any documentation

### CODE REVIEW

File: migrations/20230307200553_007.js [hunk 1]
```diff
@@ -0,0 +1,161 @@
+/**
+ * @param { import("knex").Knex } knex
+ * @returns { Promise<void> }
+ */
+exports.up = function (knex) {
+	return knex("energizers")
```
Comment (bertilnilsson) [inline migrations/20230307200553_007.js]: Is there a reason we are adding new sample data when there is already sample data created by previous migrations? Should the previous sample data be deleted first and/or updated?

File: server/api.js [hunk 2]
```diff
@@ -72,9 +74,9 @@ router.get("/energizers/:id", async (req, res) => {
 });
 
 router.post("/energizers", async (req, res) => {
-	const { name, description, rating } = req.body;
+	const { name, description, rating, url_address } = req.body;
 
-	if (!name || !description || !rating) {
+	if (!name || !description || !rating || !url_address) {
```
Comment (bertilnilsson) [inline server/api.js]: Is the url_address intended to be required?

**Comment by bertilnilsson** [comment]
Is this PR only intended to add the one column? It seems it would break the frontend since I don't think the `url_address` is being submitted with the POST?

**Comment by AzinYad** [comment]
> Is this PR only intended to add the one column? It seems it would break the frontend since I don't think the `url_address` is being submitted with the POST?
Bertil as we discussed earlier we need two columns one for image one for video address , this is what he needs help with now ,to make new push to this open PR

**Pull request merged**